### ISSUE TITLE
Resize vertical bar public in header_ FixedDataTableColumnResizerLine_ Main positioning is not accurate

### ISSUE THREAD

**Issue by qlandroid** [schrodinger/fixed-data-table-2#707]
Resize vertical bar public in header_ FixedDataTableColumnResizerLine_ Main positioning is not accurate;
![微信图片_20231124115929](https://github.com/schrodinger/fixed-data-table-2/assets/21098132/53e23033-6674-405e-ab58-7c8218dfded5)

**Comment by pradeepnschrodinger** [comment]
Hey, could you provide more details?
eg:
```
Version used:
Browser Name and version:
Operating System and version (desktop or mobile):
Link to your project:
```

Also the screenshot is blurry, so I'm not able to figure out what's exactly going wrong.

**Comment by pradeepnschrodinger** [comment]
Oh wait, I think I got the issue 🤞.

You appear to be using something like a "modal" dialog with the fixed data table rendered inside it.
The table is thus translated to a different position under a different z-index.
This somehow causes the resizer line to incorrectly render at a different position, and also get cut off by the modal.

This looks like an issue from FDT, we'll take a look at.

**Comment by pradeepnschrodinger** [comment]
@qlandroid, I've put a PR (see #709) to tackle some resizer line positioning issues. 
This should get merged in a couple of days.

However, I'm not certain if this fixes the specific issue you're facing.
It would be nice if you can post more details as per [this comment](https://github.com/schrodinger/fixed-data-table-2/issues/707#issuecomment-1825964610).

**Comment by pradeepnschrodinger** [comment]
@qlandroid , The fix for this issue is released with [v2.0.4](https://github.com/schrodinger/fixed-data-table-2/releases/tag/v2.0.4).

I'm marking the issue as closed, but feel free to continue the discussion or reopen if necessary.
